pkgs/applications/networking/mailreaders/neomutt/default.nix

```diff
@@ -15,24 +18,26 @@ let
   '';
 
 in stdenv.mkDerivation rec {
-  version = "20171208";
+  version = "20171215";
   name = "neomutt-${version}";
 
   src = fetchFromGitHub {
     owner  = "neomutt";
     repo   = "neomutt";
     rev    = "neomutt-${version}";
-    sha256 = "1fn28q4akfz0nq3ysp8n53j8yqp2mx6yhbvb59c4zm6zgd4qzgp1";
+    sha256 = "1c7vjl5cl0k41vrxp6l1sj72idz70r2rgaxa2m1yir6zb6qsrsd8";
   };
 
   buildInputs = [
     cyrus_sasl gss gpgme kerberos libidn ncurses
-    notmuch openssl perl lmdb
+    notmuch openssl perl lmdb gettext
     mime-types
-  ];
+    nullmailer # default to null, so its optional.
```

Comment:
The default is with nullmailer. `nullmailer ? null` only tells the users reading this file that it is supported to override `nullmailer` with `null`.

I don't see why this dependency should be disabled by default, only the comment is confusing.

Reply:
> nullmailer ? null only tells the users reading this file that it is supported to override nullmailer with null.

Does that mean you think ` +, withSendmail ? true, nullmailer ` would be more readable?

Reply:
I'd use
```
+, sendmail ? nullmailer, nullmailer ? null

+ buildInputs = [ … sendmail … ]
```
Then the users can disable it by overriding with `sendmail = null`, and choose another sendmail with e.g. `sendmail = msmtp`.

Reply:
BTW `sendmail` is provided by busybox, exim, msmtp, nullmailer, postfix, and ssmtp. Have you tested sending mail from `neomutt` with `nullmailer`?